foursight_core/react/api/portal_access_key_utils.py

```diff
@@ -61,7 +64,7 @@ def get_portal_access_key_info(env: str,
             if "credenti" in e or "expir" in e:
                 access_key_info["probably_expired"] = True
             access_key_info["invalid"] = True
-        access_key_info["timestamp"] = now.strftime("%Y-%m-%d %H:%M:%S")
+        access_key_info["timestamp"] = datetime_to_string(now.strftime("%Y-%m-%d %H:%M:%S"))
```

Comment:
I would like not to have any calls amid other code to something like now.strftime with a random string you could get wrong. Is there not a function that does this? can we make one?

Reply:
Oh weird, that one is just a mistake, should just be datetime_to_string(now)